src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+@import "../utils/variables";
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #ffff;
```

Comment:
1) Put repeated values into variables.
2) Either use 3 or 6 characters for color value: `#fff`  or `#ffffff`